package-lock.json

```diff
@@ -1734,7 +1734,6 @@
       "requires": {
```

Comment:
are you sure about this ... we have around 800 missing lines over here ?

Reply:
If you are asking me, will I am not the one who did that manullay 😅